leases/schema.py

```diff
@@ -41,6 +41,7 @@ class BerthLeaseNode(DjangoObjectType):
     berth = graphene.Field(BerthNode, required=True)
     status = LeaseStatusEnum(required=True)
     customer = graphene.Field("customers.schema.ProfileNode", required=True)
+    order = graphene.Field("payments.schema.OrderNode")
```

Comment:
and this? 👀

Reply:
This one causes some circular import errors, so better keep it there.